sound/soc/sof/ipc4-topology.c

```diff
@@ -1548,6 +1708,20 @@ static int sof_ipc4_widget_setup(struct snd_sof_dev *sdev, struct snd_sof_widget
 		msg = &src->msg;
 		break;
 	}
+	case snd_soc_dapm_effect:
+	{
+		struct sof_ipc4_process *process = swidget->private;
+
+		/* ipc config data size should never be 0? */
```

Comment:
Is it a question or a statement?
Will the user know _why_ this failed?

Reply:
and please print something to help identify the _why_

Reply:
OK, I will.